concourse/scripts/deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+SCRIPT_DIR=$(cd $(dirname $0) && pwd)
+
+env=${DEPLOY_ENV-$1}
+pipeline="cf-deploy"
+config="${SCRIPT_DIR}/../pipelines/deploy.yml"
+
+[[ -z "${env}" ]] && echo "Must provide environment name" && exit 100
+bash "${SCRIPT_DIR}/deploy-pipeline.sh" "${env}" "${pipeline}" "${config}"
+
+
+export ATC_URL=${ATC_URL:-"http://192.168.100.4:8080"}
+export fly_target=${FLY_TARGET:-tutorial}
+
+fly unpause-pipeline --pipeline "${pipeline}"
+curl "${ATC_URL}/pipelines/${pipeline}/jobs/init-bucket/builds" -X POST
```

Comment:
the `-b` option is the build number, for example:

```
fly -t "tutorial" watch -j cf-deploy/init-bucket -b '16'
```

The issue is that we trigger the `cf-deploy/init-bucket` but watch the next task, `cf-deploy/vpc` which we don't know the build number (it might be different).

Additionally, we need to wait until the task is triggered by  `cf-deploy/init-bucket`  to execute watch against  `cf-deploy/vpc`, if not we will get  this error:

```
$ fly -t "tutorial" watch -j cf-deploy/vpc -b '13'
2015/12/07 11:10:29 build not found
```

Solutions for this could be:

 1. Write a script to keep polling the jobs until it finishes. 
 2. Extend the command `fly` to support watch all the pipeline, instead of a specific job.